dev/drivers/scripts/plots/hurricane/jevs_hurricane_global_det_tropcyc_plots.sh

```diff
@@ -24,35 +24,36 @@ export STEP=plots
 export VERIF_CASE=tropcyc
 export envir=dev
 export cyc=00
-export job=jevs_hurricane_global_det_tropcyc_plots_${cyc}
+export job=jevs_${COMPONENT}_${RUN}_${VERIF_CASE}_${STEP}_${cyc}
 
 ############################################################
 # Load modules
 ############################################################
 module reset
 module load prod_envir/${prod_envir_ver}
-source ${HOMEevs}/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
+source ${HOMEevs}/dev/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
 
 #Set PDY to override setpdy.sh called in the j-jobs
-export PDY=20221231
+export PDY=20231231
 
 #Define the directory for TC-stats file 
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/${COMPONENT}/${RUN}/${VERIF_CASE}/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/stats/${COMPONENT}/${RUN}/${VERIF_CASE}
 #Define the directories of your NOAA/NWS logos
 export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
 
 #Define TC-vital file, and the directory for Bdeck files
-export COMINvit=/lfs/h2/emc/vpppg/noscrub/jiayi.peng/MetTCData/TCvital/syndat_tcvitals.2022
-export COMINbdeckNHC=/lfs/h2/emc/vpppg/noscrub/jiayi.peng/MetTCData/bdeck/Year2022
-export COMINbdeckJTWC=/lfs/h2/emc/vpppg/noscrub/jiayi.peng/MetTCData/bdeck/Year2022
+export COMINvit=/lfs/h2/emc/vpppg/noscrub/olivia.ostwald/Data/Year2023/TCvital/syndat_tcvitals.2023
+export COMINbdeckNHC=/lfs/h2/emc/vpppg/noscrub/olivia.ostwald/Data/Year2023/bdeck
+export COMINbdeckJTWC=/lfs/h2/emc/vpppg/noscrub/olivia.ostwald/Data/Year2023/bdeck
 
 export DATAROOT=/lfs/h2/emc/ptmp/$USER
 export COMROOT=${DATAROOT}/com
+rm -rf ${COMROOT}/evs/${evs_ver}/${STEP}/${COMPONENT}/${RUN}/${VERIF_CASE}
```

Comment:
For EE2 compliance, lets wrap this `rm -rf` inside a test.
```
if [ -d "${COMROOT}/evs/${evs_ver}/${STEP}/${COMPONENT}/${RUN}/${VERIF_CASE}" ]; then
  rm -rf ${COMROOT}/evs/${evs_ver}/${STEP}/${COMPONENT}/${RUN}/${VERIF_CASE}
fi
```
I suggest this change in any driver scripts that do the removal of the specific directory.

Reply:
Will do! I will have this revision and changing the COMIN data input paths to have $USER in my next update/"git commit/push" (unsure of the proper term)

Reply:
@OliviaOstwald-NOAA @ShelleyMelchior-NOAA I'm still trying to figure out why this would need to be removed and why this is causing problems.  This shouldn't be an issue if each individual run of any job has its own unique DATA directory, as defined in the J-job.  

It looks like you should, but what I would like to see is a test including the following line in the dev driver script:

`export jobid=$job.${PBS_JOBID:-$$}`

That currently doesn't exist now.  That would assign the Job ID number from the submitting of the PBS job and append that to the DATA directory for a unique directory.  I'd like to see if adding this line to your dev drivers would allow us to remove that line that removes an entire directory and ensures that each submission of the job has its own unique output directory.  I'm not convinced that this is the case now.

@AliciaBentley-NOAA for awareness.